README.md

```diff
@@ -237,6 +237,14 @@ There are two sets of unit tests.
  * In the solution explorer window, right Click on 'unit_tests' and select `Set as StartUp Project` so debugger will run unit tests.
  * Start a debug session.
 
+#### For Eclipse as the IDE
+ * File - new C/C++Project - Empty or Existing CMake Project
+ * Location: (`$NUPIC_CORE`) - Finish
+ * Project properties - C/C++ Build - build command set "make -C build/scripts VERBOSE=1 install -j 6"
+ * There can be issue with indexer and boost library, which can cause OS memory to overflow -> add exclude filter to
+   your project properties - Resource Filters - Exclude all folders that matches boost, recursively
```

Comment:
FYI: if you use new enough compiler/c++ standard, such as g++-8+ the project defaults to c++17 and does not need boost

Reply:
ok i will look, i don't know what version i use.. but index would be searched anyway i think.. some weird bug loops the indexer and he just stole 8GB of RAM :D